### DISCUSSION TITLE
Обводка Всс

### DISCUSSION THREAD

**Discussion by gat3v** [Sokol-Games-GmbH/PixelCombats.PublicApi#186] [Q&A]
**Будет ли обводка к оружию ВСС?**

**Comment by gat3v** [comment]
![Screenshot_20231205_223412_com pixel gun combats](https://github.com/Sokol-Games-GmbH/PixelCombats.PublicApi/assets/145331393/43077dc6-b530-4497-ad76-ac78baf2e2b9)

**Comment by kkohno** [comment]
походу забыли про нее...
